src/models/profile.js

```diff
@@ -0,0 +1,34 @@
+module.exports = (sequelize, DataTypes) => {
+    const Profile = sequelize.define("Profile", {
+        id: {
+            allowNull: false,
+            autoIncrement: true,
+            primaryKey: true,
+            type: DataTypes.INTEGER
+            },
+        firstName: {
+            type: DataTypes.STRING,
+            allowNull: true,
+            },
+        lastName: {
+            type: DataTypes.STRING,
+            allowNull: true,
+        },
+        profilePicture: {
+            type: DataTypes.STRING,
+            allowNull: true,
+        },
+        userId: {
+            type: DataTypes.INTEGER,
+            allowNull: false,
+        }
+    });
+        Profile.associate = (models) => {
```

Comment:
You do not need a profile belongs to again, since you already have an association for user, remove this...